src/pages/hotel.js

```diff
@@ -3,155 +3,138 @@ import Layout from '@components/layout'
+                ))}
+              </TableBody>
+            </Table>
+          </RoomRateTable>
 
-        <h2>How to Get Here</h2> 
-      </CenteredBox>
-      <h3>Bus</h3>
-        
-      <p>The hotel is accessible via Beach City Transit (BCT)'s route 109 southbound bus line, "LAX City Bus Center - Redondo Beach Riviera Village". The hotel is 2 minutes from the Beryl 
-        St. / Harbor Dr. stop. It is also accessible via Metro's route 232 southbound bus line, "LAX City Bus Center - Long Beach".</p>
+          <div>
+            <sup>†</sup>Taxes and fees not included in price.
+          </div>
 
-      <p>After arriving at LAX, look for the pink LAX Shuttle sign on the Lower/Arrivals Level (regardless of terminal) and board the  Metro Connector Shuttle, taking you to the LAX/Metro 
-        Transit Center. Once there, catch route 109 southbound. After about 30 minutes (27 stops), get off at the Beryl St. / Harbor Dr. stop. Walk back towards the intersection, then turn 
-        left passing Sol Restaurant. After about a 2 minute walk, the hotel will be on your left!</p>
+          <Button variant="contained" className="cta" href={bookingLink}>
+            Book online
+          </Button>
 
-      <p>Alternatively, once at the LAX/Metro Transit Center, catch route 232 southbound. After about 30 minutes (26 stops), get off at the Pacific Coast Hwy / Beryl stop. Head back 
-      towards the intersection and turn left on Beryl St. After about half a mile (about 7-10 minutes of walking), turn left on N Harbor Dr passing Sol Restaurant. After about another 2 minute 
-      walk, the hotel will be on your left!</p>
+          {bookingNote && <div>{bookingNote}</div>}
+        </CenteredBox>
 
+        <CenteredBox>
+          <ResponsiveMap>
+            <iframe title="Hotel map" maxwidth="600" height="450" style={{ border: 0 }} loading="lazy" allowFullScreen src={mapLink}></iframe>
+          </ResponsiveMap>
+          <h2>How to get here</h2> {/* I'd rather let this heading be part of body, but it wouldn't be centered */}
+        </CenteredBox>
 
-      <h3>Rideshare</h3>
-      
-      <p>Most rideshare services will take about a 30 minute drive from the airport to the hotel. Average prices range from $30-60. While we cannot directly coordinate or facilitate this, we highly encourage attendees arriving around the same time that feel comfortable to share a rideshare vehicle to help defray the cost if choosing this method.</p>
-      
-      <h3>Driving</h3>
-      
-      <p>Drive on CA-1S. CA-1S will turn slightly right and become Pacific Coast Hwy. After about 0.5 miles, turn right on N Catalina Ave, then turn right again to stay on N Catalina Ave. After another 0.5 miles, turn right onto Beryl St and then turn left onto N Harbor Dr. The hotel will be on your left! </p>
-    </PageContent>
-  </Layout>
-)}
+        {directions.map(option => (
+          <p>
+            <h3>{option.method}</h3>
+            {option.body}
```

Comment:
Unless I'm missing something this will entirely lose the newlines between paragraphs in this section. That makes this much harder to read.